src/test/flap.t.sol

```diff
@@ -150,4 +153,34 @@ contract FlapTest is DSTest {
         // check biddable
         assertTrue( Guy(ali).try_tend(id, 100 ether, 1 ether));
     }
+    function testFail_kick_over_lid() public {
+        flap.kick({ lot: 501 ether
+                  , bid: 0
+                  });
+    }
+    function testFail_kick_over_lid_2_auctions() public {
+        // Just up to the lid
+        flap.kick({ lot: 500 ether
+                  , bid: 0
+                  });
+        // Just over the lid
+        flap.kick({ lot: 1
+                  , bid: 0
+                  });
+    }
+    function test_deal() public {
```

Comment:
Can you add another test very similar to this that has more than one auction running at once and that asserts the intermediate state?

Reply:
This is done.